src/WebApiContrib.Core.Formatter.Csv/project.json

```diff
@@ -0,0 +1,21 @@
+{
+  "version": "1.0.0-*",
+
+    "dependencies": {
+        "Microsoft.AspNetCore.Mvc.Core": "1.0.0-rc2-final",
+        "NETStandard.Library": "1.5.0-rc2-24027",
+        "System.Reflection.TypeExtensions": "4.1.0-rc2-24027"
+    },
+
+    "frameworks": {
+        "netstandard1.5": {
+            "imports": "dnxcore50"
+        },
+        "net46": {

```

Comment:
Why net451, this is not required for the standard ASP.NET Core web project. If I need to support net451, I would use the existing Web API libraries